low-level/framework/base/sspl_constants.py

```diff
@@ -33,13 +32,13 @@
 cs_products = ["CS-A"]
 cs_legacy_products = ["CS-L", "CS-G"]
 setups = ["vm", "cortx", "ssu", "gw", "cmu"]
-RESOURCE_PATH = "/opt/seagate/%s/sspl/low-level/json_msgs/schemas/" % (PRODUCT_FAMILY)
-CLI_RESOURCE_PATH = "/opt/seagate/%s/sspl/low-level/tests/manual" % (PRODUCT_FAMILY)
-DATA_PATH = "/var/%s/sspl/data/" % (PRODUCT_FAMILY)
-SSPL_CONFIGURED_DIR = "/var/%s/sspl" % (PRODUCT_FAMILY)
+RESOURCE_PATH = "/opt/seagate/%s/sspl/low-level/json_msgs/schemas/" % PRODUCT_FAMILY
```

Comment:
Not sure why these changes even, existing ones are more readable anyways

Reply:
Just changed these lines to avoid parenthesis. Added back as you feel readability is better with parenthesis.

Reply:
Done.